grafana-plugin/src/containers/EscalationVariants/EscalationVariants.tsx

```diff
@@ -170,7 +157,17 @@ const EscalationVariants = observer(
             onUserSelect={(user: User) => {
               onUpdateEscalationVariants({
                 ...value,
-                userResponders: [...value.userResponders, { type: ResponderType.User, data: user, important: false }],
+                userResponders: [
+                  ...value.userResponders,
+                  {
+                    type: ResponderType.User,
+                    data: user,
+                    important:
+                      user.notification_chain_verbal.important && !user.notification_chain_verbal.default
+                        ? true
```

Comment:
Can't use just the boolean check instead of the ternary?

`!!(user.notification_chain_verbal.important && !user.notification_chain_verbal.default)` 

or

`Boolean(user.notification_chain_verbal.important && !user.notification_chain_verbal.default)`